mockwebserver/src/test/java/okhttp3/mockwebserver/RecordedRequestTest.java

```diff
@@ -102,4 +102,14 @@ private FakeSocket(InetAddress localAddress, int localPort, InetAddress remoteAd
 
     assertThat(request.getRequestUrl().toString()).isEqualTo("http://127.0.0.1/");
   }
+
+  @Test public void testUsesLocalIpv6WithoutBrackets() throws UnknownHostException {
```

Comment:
👍 

nit: we might add a test for IPv4 as well? ;)
e.g. 
```
  @Test public void testLocalhostIpv4() throws UnknownHostException {
    Socket socket =
        new FakeSocket(InetAddress.getByAddress("localhost", new byte[] {127, 0, 0, 1}), 80);

    RecordedRequest request = new RecordedRequest("GET / HTTP/1.1", headers,
        Collections.emptyList(), 0, new Buffer(), 0, socket);

    assertThat(request.getRequestUrl().toString()).isEqualTo("http://localhost/");
  }
```

Reply:
Thanks!